python/delta_sharing/delta_sharing.py

```diff
@@ -51,6 +51,53 @@ def _parse_url(url: str) -> Tuple[str, str, str, str]:
     return (profile, share, schema, table)
 
 
+def get_table_version(
+    url: str,
+    starting_timestamp: Optional[str] = None
+) -> int:
+    """
+    Get the shared table version using the given url.
+
+    :param url: a url under the format "<profile>#<share>.<schema>.<table>"
+    :param starting_timestamp: a string in the format of YYYY-MM-DDThh:mm:ssZ. Get the version at or
+      after the given timestamp.
```

Comment:
Perhaps add that if this is not specified we get the latest version?

Reply:
added